code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -443,7 +443,7 @@
 		msg += "<span class='warning'><b>[src] has \a [implant] sticking out of [t_his] flesh!</b></span>\n"
 	if(digitalcamo)
 		msg += "[t_He] [t_is] repulsively uncanny!\n"
-	if(!wear_mask && is_thrall(src) && in_range(user,src))
+	if(!(wear_mask.flags_inv & HIDEFACE || wear_mask.flags_cover & MASKCOVERSMOUTH) && is_thrall(src) && in_range(user,src))

```

Comment:
Might want to check for `skipface` here, so you don't magically see through their hardsuit helmet.